### ISSUE TITLE
History graphs not refreshing at all

### ISSUE THREAD

**Issue by OrbitingOcelot** [home-assistant/frontend#6520]
## Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.

## The problem

My lovelace UI history graph cards do not get refreshed. Never. The information is shown correctly, but it is never updated, only if I reload the page manually. Once the page is refreshed manually by pressing F5, the timeline is updated and the information shown correctly.

## Expected behavior

UI history graph cards should update periodically while the web page is open.

## Steps to reproduce

1. Download clean hass.io VDI image and install it in virtualbox.
2. Add a history graph card (optionally remove everything else from the UI).
4. Exit UI edit mode. Optionally reload the page.
5. Wait for refresh of the card -> never happens.
6. Hit F5 to reload page - new info appears.

## Environment

- Home Assistant release with the issue: Home Assistant 0.113.3
- Last working Home Assistant release (if known): not known
- Browser and browser version: Chrome 84.0.4147.105, Firefox 79.0. All add-ons disabled. Cache cleared.
- Operating system: Windows 10 Home 64 Bit

## State of relevant entities

sensor.compal_broadband_networks_inc_ch7465lg_packets_s_sent | 2.1 | unit_of_measurement: packets/s friendly_name: Compal Broadband Networks, Inc CH7465LG packets/s sent icon: mdi:server-network
-- | -- | --

## Problem-relevant configuration

Home UI reduced to two graphs to highlight the problem and rule out other issues:

```yaml
title: Home
views:
  - path: default_view
    title: Home
    badges: null
    cards:
      - type: history-graph
        entities:
          - entity: sun.sun
        hours_to_show: 1
        refresh_interval: 5
      - type: history-graph
        entities:
          - entity: sensor.compal_broadband_networks_inc_ch7465lg_packets_s_sent
        hours_to_show: 1
        refresh_interval: 5
```

## Javascript errors shown in your browser console/inspector

```txt
n/a
```

## Additional information

sis | vm
-- | --
dev | false
docker | true
docker_version | 19.03.8
hassio | true
host_os | HassOS 4.11
installation_type | Home Assistant OS
os_name | Linux
os_version | 5.4.50
python_version | 3.8.3
supervisor | 229
timezone | UTC
version | 0.113.3
virtualenv | false

**Comment by OrbitingOcelot** [comment]
I have also looked at this, which might be related: https://github.com/home-assistant/frontend/issues/6135

However my issue is different - I do not need to go to config ui mode, my refresh simply never works at all, even if I freshly login without touching anything. Therefore I have opted to open a separate issue. Thank you very much.

**Comment by mfabiani53** [comment]
It seems i am having the same problem, but not related only to history graphs but to all the refreshings in the UI.
I discovered it first  looking at the media players not refreshing artwork and title track when playing some music on spotify. They refresh only after i do a reload and everytime i do a reload i get infos about a change in the ui_lovelace.yaml also if i don't change anything. More, i have the same issues when a card change his state forcing me to reload to have the state change. I noticed this issue also when updating a component with hacs. I get the 'update' button always running also if the component is already updated and i need to close the window to go back to the hacs frontend. I discovered these issues independently from the browser i am using and from the PC where to run the browser. I tried also to clear the caches but nothing was improved. It begin to happen just after upgrading to HA 0.113.
If You all think i need to open a separate issue, just let me know, but i hope it will be fixed asap.

**Comment by OrbitingOcelot** [comment]
The rest of my UI actually works and refreshes OK. If I turn on a light using a button, I see the state of the light changing in the UI. The only thing that does not refresh are my history graphs.
@mfabiani53 Your issue seems to be wider and may be unrelated, so I believe you should open a separate issue describing your specific situation.

**Comment by mfabiani53** [comment]
Ok, thanks @OrbitingOcelot I'll do it.

**Comment by carealtor** [comment]
I'm having the same issue.

**Comment by bramkragten** [comment]
Fixed in https://github.com/home-assistant/frontend/pull/6647
